### DISCUSSION TITLE
有办法获取某个指定的进程发出的http请求呢

### DISCUSSION THREAD

**Discussion by toohandsome** [gojue/ecapture#328] [General]
有办法获取某个指定的进程发出的http请求呢，目前看起来都是ssl，普通的http 请求有办法获取到吗

**Comment by cfc4n** [comment]
eCapture项目核心目标就是支持`https/tls`的明文捕获，可以指定进程PID，详情看help参数。 但不支持`http`的通讯捕获功能。

**Comment by Your7Maxx** [comment]
关于http请求关联进程的实践可以关注下这个项目：https://github.com/Your7Maxx/FlowGod
